File: Water Bowls/water.py
with open("water.in", "r") as input_file:
    input_list = [line.strip() for line in input_file]
with open("water.out", "w") as output_file:
    for binary_diameter in input_list:
        print(round((2/3 * 3.14 * (int(binary_diameter, 2) ** 3)) / 1000), file=output_file)

File: Water Bowls/water.cpp
#include <iostream>
#include <fstream>
#include <cmath>

using namespace std;

int main() {
    fstream input("water.in", ios::in);
    fstream output("water.out", ios::out);

    for (string line; getline(input, line);) {
        output << round(2.0/3 * 3.14 * pow(stoi(line, 0, 2), 3) / 1000) << endl;
    }

    output.close();
    input.close();

    return 0;
}
